### ISSUE TITLE
Invalid OpenAPI spec

### ISSUE THREAD

**Issue by nmeyerhans** [firecracker-microvm/firecracker#774]
`api_server/swagger/firecracker.yaml` does not validate as an [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) spec.

```
firecracker‹master›$ docker run --rm --net=none -v $PWD:/work -w /work quay.io/goswagger/swagger validate  api_server/swagger/firecracker.yaml

The swagger spec at "api_server/swagger/firecracker.yaml" is invalid against swagger specification 2.0.
See errors below:
- "definitions.Logger.properties.options.items" must validate at least one schema (anyOf)
- definitions.Logger.properties.options.items in body must be of type object: "string"
- info.contact.email in body must be of type email: "compute-capsule@amazon.com"
```

In particular, the `definitions.Logger.properties.options.items` issue is causing the Go swagger generator to crash. Per the spec, an array's "items" field must reference an [Items Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#itemsObject), not a string.

**Comment by nmeyerhans** [comment]
Note that the validator's complaints about the email address are a red-herring. The validator is using network resources to try to determine the validity of compute-capsule@amazon.com, but I've disabled networking in the docker invocation, so the validation fails.

**Comment by aghecenco** [comment]
Closed by #775
